README.md

```diff
@@ -116,6 +116,12 @@ To see local examples in action, do this:
 npm install
 npm run examples
 ```
+or to see the examples running in an isomorphic environment:
```

Comment:
lets use the term "universal" instead of "isomorphic", or even just "server side rendering" instead